### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by xumeng723**
Add 5 examples in the readme file ,added some links in the file.

### CODE REVIEW

File: examples/readme.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+
+
+* different training platforms ： "local", "remote", "pai", "kubeflow" <br>
+
+
+|type|item name|code directory|
+|---|---|---|
+|local<br>remote<br>pai<br>kubeflow|[MNIST with NNI API](#mnist)<br>[distributed MNIST (tensorflow) using kubeflow](#mnist-kubeflow-tf)<br>[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|[examples/trials/mnist/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist)<br>[examples/trials/mnist-distributed/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed)<br>[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)
```
Comment (PurityFan) [inline examples/readme.md]: Maybe here can't show the difference clearly. Can you consider list the examples for "local", "remote", "pai", "kubeflow" respectively ?

File: examples/readme.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+NNI offers a variety of features to support different search space modes, different training platforms, different tuners and different deeplearning frameworks. The goal of this tutorial is to summarize the different features that make it easy for users to start the task that suits them.
+
+We have selected the following common aspects:
```
Comment (PurityFan) [inline examples/readme.md]: You can add a navigation bar that can jump quickly to the specific part from here. For example:
* different search space modes: "api", "annotation"
* different training platforms: "local", "remote", "pai", "kubeflow"
* different tuners: "tpe", "batch-tune", "hyperband"
* ..........

File: examples/readme.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+* different search space modes ： "api", "annotation" <br>
```
Comment (PurityFan) [inline examples/readme.md]: Unified use of the English colon, is ": ", not “：”

File: examples/readme.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+
+
+* different training platforms ： "local", "remote", "pai", "kubeflow" <br>
```
Comment (PurityFan) [inline examples/readme.md]: same here, ":"

File: examples/readme.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+|local<br>remote<br>pai|[MNIST with NNI annotation](#mnist-annotation)<br>[MNIST in keras](#mnist-keras)<br>[MNIST -- tuning with batch tuner](#mnist-batch)<br>[MNIST -- tuning with hyperband](#mnist-hyperband)<br>[MNIST -- tuning within a nested search space](#mnist-nested)|[examples/trials/mnist-annotation/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-annotation)<br>[examples/trials/mnist-keras/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-keras)<br>[examples/trials/mnist-batch-tune-keras](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-batch-tune-keras)<br>[examples/trials/mnist-hyperband/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-hyperband)<br>[examples/trials/mnist-cascading-search-space/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-cascading-search-space)
+
+
+
+* different tuners ："tpe", "batch-tune", "hyperband" <br>
```
Comment (PurityFan) [inline examples/readme.md]: same here, ":"

File: examples/readme.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+|tpe|[MNIST with NNI API](#mnist)<br>[MNIST with NNI annotation](#mnist-annotation)<br>[MNIST in keras](#mnist-keras)<br>[MNIST -- tuning within a nested search space](#mnist-nested)<br>[distributed MNIST (tensorflow) using kubeflow](#mnist-kubeflow-tf)<br>[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|[examples/trials/mnist/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist)<br>[examples/trials/mnist-annotation/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-annotation)<br>[examples/trials/mnist-keras/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-keras)<br>[examples/trials/mnist-cascading-search-space/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-cascading-search-space)<br>[examples/trials/mnist-distributed/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed)<br>[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)|
+
+
+
+* different deeplearning frameworks ： "tensorflow", "pytorch", "keras"
```
Comment (PurityFan) [inline examples/readme.md]: same here, ":"

File: examples/readme.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+|pytorch|[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)|
+|keras|[MNIST in keras](#mnist-keras)<br>[MNIST -- tuning with batch tuner](#mnist-batch)|[examples/trials/mnist-keras/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-keras)<br>
+
+We also have some other examples :<br>
+<br><br>[CIFAR-10 examples](https://github.com/Microsoft/nni/tree/master/examples/trials/cifar10_pytorch):CIFAR-10 classification is a common benchmark problem in machine learning. The CIFAR-10 dataset is the collection of images. It is one of the most widely used datasets for machine learning research which contains 60,000 32x32 color images in 10 different classes. Thus, we use CIFAR-10 classification as an example to introduce NNI usage.
```
Comment (PurityFan) [inline examples/readme.md]:
```suggestion
<br><br>[CIFAR-10 examples](https://github.com/Microsoft/nni/tree/master/examples/trials/cifar10_pytorch): CIFAR-10 classification is a common benchmark problem in machine learning. The CIFAR-10 dataset is the collection of images. It is one of the most widely used datasets for machine learning research which contains 60,000 32x32 color images in 10 different classes. Thus, we use CIFAR-10 classification as an example to introduce NNI usage.
```

Comment (PurityFan) [reply examples/readme.md]: Add a space after the colon

File: examples/readme.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+<br><br>[Scikit-learn in NNI](https://github.com/Microsoft/nni/tree/master/examples/trials/sklearn):Scikit-learn is a pupular meachine learning tool for data mining and data analysis. It supports many kinds of meachine learning models like LinearRegression, LogisticRegression, DecisionTree, SVM etc. How to make the use of scikit-learn more efficiency is a valuable topic.NNI supports many kinds of tuning algorithms to search the best models and/or hyper-parameters for scikit-learn, and support many kinds of environments like local machine, remote servers and cloud.
```
Comment (PurityFan) [inline examples/readme.md]: Add a space after the colon， same here

File: examples/readme.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+<br><br>[Automatic Model Architecture Search for Reading Comprehension](https://github.com/Microsoft/nni/tree/master/examples/trials/network_morphism):This example shows us how to use Genetic Algorithm to find good model architectures for Reading Comprehension.
```
Comment (PurityFan) [inline examples/readme.md]: Add a space after the colon， same here

File: examples/readme.md [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+<br><br>[GBDT in nni](https://github.com/Microsoft/nni/tree/master/examples/trials/auto-gbdt):Gradient boosting is a machine learning technique for regression and classification problems, which produces a prediction model in the form of an ensemble of weak prediction models, typically decision trees. It builds the model in a stage-wise fashion as other boosting methods do, and it generalizes them by allowing optimization of an arbitrary differentiable loss function.
```
Comment (PurityFan) [inline examples/readme.md]: Add a space after the colon, same here

File: examples/readme.md [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+|[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|json|local,remote,pai,kubeflow|pytorch|tpe|[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)
+
+
+
+<a name ="mnist"></a>
+**MNIST with NNI API**
+
+This is a simple network which has two convolutional layers, two pooling layers and a fully connected layer. We tune hyperparameters, such as dropout rate, convolution size, hidden size, etc. It can be tuned with most NNI built-in tuners, such as TPE, SMAC, Random. We also provide an exmaple yaml file which enables assessor.
+
+As we stated in the target, we target to find out the best `optimizer` for training . When using different optimizers, we also need to adjust `dropout_rate` ,`conv_size`,`hidden_size`,`batch_size`and `learning_rate` accordingly. so we chose these three parameters as hyperparameters and write the following search space.
```
Comment (xuehui1991) [inline examples/readme.md]: "we target to find out the best `optimizer` for training"????

File: examples/readme.md [hunk 12]
```diff
@@ -0,0 +1,151 @@
+NNI offers a variety of features to support different search space modes, different training platforms, different tuners and different deeplearning frameworks. The goal of this tutorial is to summarize the different features that make it easy for users to start the task that suits them.
```
Comment (xuehui1991) [inline examples/readme.md]: README.md

File: examples/readme.md (hunk 10 above)
Comment (xuehui1991) [inline examples/readme.md]: "GBDT in nni" ---》 “GBDT in NNI”？ Due to you use "Scikit-learn in NNI" in line 44.

File: examples/readme.md (hunk 4 above)
Comment (scarlett2018) [inline examples/readme.md]: pls add a reference link for PAI project.

**Pull request closed without merging**